org.eclipse.january.test/src/org/eclipse/january/metadata/MetadataProviderTest.java

```diff
@@ -32,82 +29,10 @@ public void testGetMetadata() throws MetadataException {
 		mdp.clearMetadata(null);
 
 		// add sub-class of IMetadata
-		mdp.addMetadata(new InnerMetadata());
+		mdp.addMetadata(new MetadataTest.InnerMetadata());
 		System.err.println(mdp.getMetadata());
 		Assert.assertTrue(mdp.getMetadata() instanceof IMetadata);
```

Comment:
@PeterC-DLS is there merit in either switching the test to use the preferred way of getting metadata (````getFirstMetadata()````, rather than ````getMetadata()````), or adding an additional method to test the preferred method?